lib/rouge/lexers/solidity.rb

```diff
@@ -96,7 +96,8 @@ def self.reserved
       state :inline_whitespace do
         rule %r/[ \t\r]+/, Text
         rule %r/\\\n/, Text # line continuation
-        rule %r(/(\\\n)?[*].*?[*](\\\n)?/)m, Comment::Multiline
+        rule %r(/[*].*?[*]/)m, Comment::Multiline
+        rule %r(/\*.*(\*/){0})m, Comment::Multiline # open to EOF
```

Comment:
I'm a bit lost at the purpose of `(\*/){0}`. Why is that there?

Reply:
Oh wait—sorry, I just saw the discussion to the previous comment. I'm looking at that together with this now.